plover/gui/config.py

```diff
@@ -476,15 +479,29 @@ def __init__(self, config, parent):
         sizer.Add(self.show_strokes, border=UI_BORDER, 
                   flag=wx.LEFT | wx.RIGHT | wx.BOTTOM)
 
+        show_suggestions_button = wx.Button(self,
+                                        label=self.SHOW_SUGGESTIONS_BUTTON_TEXT)
+        show_suggestions_button.Bind(wx.EVT_BUTTON, self.on_show_suggestions)
+        sizer.Add(show_suggestions_button, border=UI_BORDER, flag=wx.ALL)
+
+        self.show_suggestions = wx.CheckBox(self, label=self.SHOW_SUGGESTIONS_TEXT)
+        self.show_suggestions.SetValue(config.get_show_suggestions_display())
+        sizer.Add(self.show_suggestions, border=UI_BORDER,
+                  flag=wx.LEFT | wx.RIGHT | wx.BOTTOM)
+
         self.SetSizer(sizer)
 
     def save(self):
         """Write all parameters to the config."""
         self.config.set_show_stroke_display(self.show_strokes.GetValue())
+        self.config.set_show_suggestions_display(self.show_suggestions.GetValue())
 
     def on_show_strokes(self, event):
         StrokeDisplayDialog.display(self.GetParent(), self.config)
 
+    def on_show_suggestions(self, event):
+        SuggestionsDisplayDialog.display(self.GetParent(), self.config, self.engine)
```

Comment:
Hi Balthamos.

I'm getting an error on this line.

'DisplayConfig' object has no attribute 'engine'

Thanks
Mike